### PR TITLE
updated EventHub.json as per review comments

### PR DESCRIPTION
**Pull request by v-Ajnava**

### CODE REVIEW

File: arm-servicebus/2015-08-01/swagger/EventHub.json [hunk 1]
```diff
@@ -0,0 +1,1791 @@
...
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}": {
+      "put": {
+        "tags": [
+          "Namespaces"
+        ],
+        "operationId": "Namespaces_CreateOrUpdate",
+        "description": "Creates/Updates a service namespace. Once created, this namespace's resource manifest is immutable. This operation is idempotent.",
```
Comment (sazeesha) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Creates/Updates a EventHub namespace.

File: arm-servicebus/2015-08-01/swagger/EventHub.json [hunk 2]
```diff
@@ -0,0 +1,1791 @@
...
+          "type": "string",
+          "description": "ARM type of the namespace."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the Event Hub."
+        },
+        "properties": {
+          "$ref": "#/definitions/EventHubProperties",
+		  "$ref": "#/definitions/TopicProperties"
```
Comment (sazeesha) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Remove this

File: arm-servicebus/2015-08-01/swagger/EventHub.json [hunk 3]
```diff
@@ -0,0 +1,1791 @@
...
+		  "items": {
+		  "type": "integer"
+		  },
+          "description": "Current number of shards on the Event Hub."
+        },
+        "status": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of the EventHub.",
+          "enum": [
+            "Active",
```
Comment (sazeesha) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Fix tabbing.

In general fix it everywhere. You can use VS for that.

File: arm-servicebus/2015-08-01/swagger/EventHub.json (hunk 1 above)
Comment (v-Ajnava) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Resolved

File: arm-servicebus/2015-08-01/swagger/EventHub.json (hunk 2 above)
Comment (v-Ajnava) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Resolved

File: arm-servicebus/2015-08-01/swagger/EventHub.json (hunk 3 above)
Comment (v-Ajnava) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: Resolved

File: arm-servicebus/2015-08-01/swagger/EventHub.json [hunk 4]
```diff
@@ -0,0 +1,1790 @@
...
+              "$ref": "#/definitions/NamespaceListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceBus/namespaces": {
```
Comment (sazeesha) [inline arm-servicebus/2015-08-01/swagger/EventHub.json]: EH

**Pull request merged**